src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,15 +1,41 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
```

Comment:
Some of the most readable code that I have seen. 

I think I made mistake with naming parameters as ArrA and ArrB

I suggest we stick python with snake case and name them instead arr_left and arr_right respectively.